pom.xml

```diff
@@ -49,17 +53,44 @@
     <module>quickstart</module>
     <module>documentation</module>
   </modules>
+  <dependencyManagement>
+    <dependencies>
```

Comment:
This is IMO the best place to declare test dependencies that we don't want to clutter the BOMs with.